content/developers/dogstatsd/unix_socket.md

```diff
@@ -0,0 +1,122 @@
+---
+title: DogStatsD over Unix Domain Socket
+kind: documentation
+description: "Usage documentation for DogStatsD over Unix Domain Sockets".
+further_reading:
+- link: "developers/dogstatsd"
+  tag: "Documentation"
+  text: "Introduction to DogStatsD"
+- link: "developers/libraries"
+  tag: "Documentation"
+  text: "Official and Community-contributed API and DogStatsD client libraries"
+- link: "https://github.com/DataDog/datadog-agent/tree/master/pkg/dogstatsd"
+  tag: "Github"
+  text: "DogStatsD source code"
+---
+
+Starting with version 6.0, the Datadog Agent is able to ingest metrics via a Unix Domain Socket (UDS), as an alternative to UDP, when running on Linux systems.
+
+While UDP works great on `localhost`, it can be a challenge to setup in containerized environments. Unix Domain Sockets allow you to easily establish the connection via a socket file, regardless of the IP of the Datadog Agent container. It also enables the following benefits:
+
+* Bypassing the networking stack brings a significant performance improvement for high traffic
+* While UDP has no error handling, UDS allows the Agent to detect dropped packets and connection errors, while still allowing a non-blocking use
+* DogStatsD is able to detect the origin container of metrics and tag them accordingly
```

Comment:
"Them" in this sentence is ambiguous, could refer to the container or the metrics.

Some options:
"DogStatsD is able to detect the origin container of metrics and tag those metrics accordingly," or
"DogStatsD is able to detect the container from which metrics originated, and tag those metrics accordingly," or
"DogStatsD is able to tag metrics with their origin container." (If tagging is possible then detection is implied.)

Reply:
ping @xvello

Reply:
Reworded, thanks for providing options 👍